dev/AutoSuggestBox/AutoSuggestBox_themeresources.xaml

```diff
@@ -426,6 +429,7 @@
                                     x:Name="SuggestionsList"
                                     DisplayMemberPath="{TemplateBinding DisplayMemberPath}"
                                     IsItemClickEnabled="True"
+                                    FontSize="{TemplateBinding FontSize}"
```

Comment:
Why only size and not the other font properties?

Reply:
For some reason, this is the only template binding we need to apply in that area of the code. The other font settings get passed doen correctly.